### PR TITLE
Example dashboard, React + TypeScript + Tailwind CSS

### PR DESCRIPTION
**Pull request by suraffy**

### CODE REVIEW

File: example-dashboard/README.md
Comment (agebeyaw) [inline example-dashboard/README.md:1]: Update the readme with contextual guide on how to run the example project extracting only the relevant content from the auto generated readme.

File: example-dashboard/src/components/Main.tsx
Comment (agebeyaw) [inline example-dashboard/src/components/Main.tsx:1]: I know this is temporary, but you may put some meaningful text

File: example-dashboard/src/components/common/Sidebar.tsx
Comment (agebeyaw) [inline example-dashboard/src/components/common/Sidebar.tsx:1]: Add the sidebar menu items when they are ready, imagine there is a QA person that tests implemented features. Let's include things quite ready in the PR

**Review by agebeyaw — APPROVED** [review]

**Review by suraffy — COMMENTED** [review]
noted

**Review by ermiyas111 — APPROVED** [review]

**Pull request closed without merging**